versionist.conf.js

```diff
@@ -1,32 +1,59 @@
-module.exports = {
+var _ = require('lodash');
+var execSync = require('child_process').execSync;
 
-  subjectParser: 'angular',
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
```

Comment:
Personally, I don't think so. Given a commit hash, users of `versionist` can devise however they want to use it.
Others may well want just the email address, or both name and email address, or the time of the commit, or... where would we stop?
Making case flexible is more binary.